Code/SimDivPickers/Wrap/MaxMinPicker.cpp

```diff
@@ -57,27 +56,14 @@ RDKit::INT_VECT MaxMinPicks(MaxMinPicker *picker, python::object distMat,
 
 class pyobjFunctor {
  public:
-  pyobjFunctor(python::object obj, bool useCache)
-      : dp_obj(obj), dp_cache(NULL) {
-    if (useCache)
-      dp_cache = new std::map<std::pair<unsigned int, unsigned int>, double>();
-  }
-  ~pyobjFunctor() { delete dp_cache; }
+  pyobjFunctor(python::object obj) : dp_obj(obj) {}
+  ~pyobjFunctor() {}
   double operator()(unsigned int i, unsigned int j) {
-    double res;
-    std::pair<unsigned int, unsigned int> idxPair(i, j);
-    if (dp_cache && dp_cache->count(idxPair) > 0) {
-      res = (*dp_cache)[idxPair];
-    } else {
-      res = python::extract<double>(dp_obj(i, j));
-      if (dp_cache) (*dp_cache)[idxPair] = res;
-    }
-    return res;
+    return python::extract<double>(dp_obj(i, j));
   }
 
  private:
   python::object dp_obj;
-  std::map<std::pair<unsigned int, unsigned int>, double> *dp_cache;
 };
 
 RDKit::INT_VECT LazyMaxMinPicks(MaxMinPicker *picker, python::object distFunc,
```

Comment:
Should we add a deprecated message to useCache so we can remove it later?

Reply:
done